packages/common/src/webrtc/helpers.ts

```diff
@@ -290,6 +290,23 @@ const sdpBitrateHack = (sdp: string, max: number, min: number, start: number) =>
   return lines.join(endOfLine)
 }
 
+const filterIceServers = (servers: RTCIceServer[], disableUDP= false): RTCIceServer[] => {
```

Comment:
I would suggest setting the second argument as an object.

For eg:
```js
{ disableUDP: true } 
```

So that later we can pass other values such as `disableTCP` for filtering the servers.